### ISSUE TITLE
When doing PR, emails come from github

### ISSUE THREAD

**Issue by stevesmename** [bradwestfall/webdevphoenix#17]
when a user is contributing a company they use the Github interface for convenience. I personally forked the project and created a company page which will later be requested as a pull. Forking the project then receives the following emails. 

```
The page build completed successfully, but returned the following warning:

CNAME already taken: webdevphoenix.com

For information on troubleshooting Jekyll see:

  https://help.github.com/articles/using-jekyll-with-pages#troubleshooting

If you have any questions you can contact us by replying to this email. 
```

**Comment by bradwestfall** [comment]
Hmm, yea that's interesting. So I'm not sure how familiar you are with github pages, but the idea is that if you have a repo with a "gh-pages" branch, then that branch automatically gets turned into a public facing website. Having a file called CNAME in that branch tells GitHub that you intend to point a domain name to GitHub. So it's somewhat of a race, the first repo to claim a domain name by virtue of having a CNAME file will create this error message for other repos on github that try to claim the same name. 

So that's how it works. I don't really know what we can do about it. Unless it is possible to only fork the master branch and work there? Then do a pull request to the webdevphoenix master branch. If that were possible, github wouldn't complain about the CNAME file in a master branch, only a gh-pages branch

**Comment by stevesmename** [comment]
Yep, the good news is that the email doesn't keep populating my inbox. I received only two in the first hour and no further emails. A suggested fix might be to do a git submodule for the 'companies' directory, allowing a separate queue for PR of bugs (this repo) and PR of company updates (git submodule repo). 

OR update contrib.md with the preferred method to add a PR and/or just list this as a known issue. Either way with this issue reported, others won't likely report the same issue.

**Comment by stevesmename** [comment]
#20
